release_dashboard/lib/widgets/conductor_status.dart

```diff
@@ -79,18 +76,18 @@ class ConductorStatusState extends State<ConductorStatus> {
         Wrap(
           children: <Widget>[
             Column(
-              children: <Widget>[
-                RepoInfoExpansion(engineOrFramework: 'engine', releaseStatus: releaseStatus!),
-                const SizedBox(height: 10.0),
-                CherrypickTable(engineOrFramework: 'engine', releaseStatus: releaseStatus),
+              children: const <Widget>[
+                RepoInfoExpansion(engineOrFramework: 'engine'),
```

Comment:
This will be converted from string to enums as well in this PR: https://github.com/flutter/cocoon/pull/1477